PROVESFlightControllerReference/test/int/rtc_test.py

```
@@ -333,12 +344,45 @@ def test_06_rtc_alarm_cancellation(fprime_test_api: IntegrationTestAPI, start_gd
+    assert alarm_id is not None, "Failed to obtain alarm id from AlarmSet event"
+
+    # Cancel the alarm by ID
+    fprime_test_api.send_command(f"{rtcManager}.ALARM_CANCEL", [alarm_id])
+
+    # Assert AlarmCanceled references the same ID
+    alarm_canceled_evt: EventData = fprime_test_api.assert_event(
+        f"{rtcManager}.AlarmCanceled", start=start, timeout=5
+    )
+    assert alarm_canceled_evt.args and alarm_canceled_evt.args[0].val == alarm_id, (
+        f"AlarmCanceled id {alarm_canceled_evt.args[0].val} did not match expected {alarm_id}"
+    )
```

Ruff (0.16.1)

[warning] Assertion should be broken down into multiple parts

(PT018)